cachito/web/models.py

```diff
@@ -382,19 +389,19 @@ def to_json(self, verbose=True):
                 mapping.dependency_id: replacement_id_to_replacement[mapping.replaced_dependency_id]
                 for mapping in self.replaced_dependency_mappings
             }
-            rv.update({'dependencies': [
+            rv['dependencies'] = [
                 dep.to_json(dep_id_to_replacement.get(dep.id), force_replaces=True)
                 for dep in self.dependencies
-            ]})
+            ]
         else:
-            rv.update({'dependencies': self.dependencies_count})
+            rv['dependencies'] = self.dependencies_count
         return rv
 
     @classmethod
     def from_json(cls, kwargs):
         # Validate all required parameters are present
         required_params = {'repo', 'ref'}
-        optional_params = {'dependency_replacements', 'flags', 'pkg_managers'}
+        optional_params = {'dependency_replacements', 'flags', 'pkg_managers', 'user'}
```

Comment:
Keeping the param name as `user` might be confusing. Should we make it more specific like `proxy_username`?

Reply:
I chose `user` because that is the key used on the JSON output. I thought it'd be good to be consistent in that regard.

I'm not opposed to changing it but would like @lcarva's opinion on this.